integration-tests/file/src/test/java/org/apache/camel/quarkus/component/file/it/FileTest.java

```diff
@@ -99,36 +200,78 @@ public void fileReadLock_minLength() throws Exception {
         Thread.sleep(10_000L);
 
         // Read the file that should not be there
-        RestAssured
-                .get("/file/get/{folder}/{name}", FileRoutes.READ_LOCK_OUT, Paths.get(fileName).getFileName())
+        RestAssured.given()
+                .post("/file/get/{folder}/{name}", FileRoutes.READ_LOCK_OUT, Paths.get(fileName).getFileName())
                 .then()
                 .statusCode(204);
     }
 
     @Test
     public void quartzSchedulerFilePollingConsumer() throws InterruptedException {
-        String fileName = RestAssured.given()
-                .contentType(ContentType.TEXT)
-                .body(FILE_BODY)
-                .post("/file/create/quartz")
-                .then()
-                .statusCode(201)
-                .extract()
-                .body()
-                .asString();
+        String fileName = createFile(FILE_BODY, "/file/create/quartz");
 
         String targetFileName = Paths.get(fileName).toFile().getName();
         await().atMost(10, TimeUnit.SECONDS).until(() -> {
             return Files.exists(Paths.get("target/quartz/out", targetFileName));
         });
 
         RestAssured
-                .get("/file/get/{folder}/{name}", "quartz/out", targetFileName)
+                .given()
+                .post("/file/get/{folder}/{name}", "quartz/out", targetFileName)
                 .then()
                 .statusCode(200)
                 .body(equalTo(FILE_BODY));
     }
 
+    private static String createFile(String content, String path) {
+        return createFile(content.getBytes(), path, null, null);
```

Comment:
String.getBytes() uses the platform default encoding - I guess there are platforms there in the wild having strange default encodings that might make these tests fail. So it is perhaps better to pass an explicit encoding, e.g. String.getBytes(StandardEncodings.UTF_8) to make the test robust across platforms and locales.

Reply:
Totally makes sense, I'll do it

Reply:
done